### PR TITLE
Remove dependency upon internet URLs for selenium tests

### PR DESCRIPTION
**Pull request by rnorth**
Using the new shiny https://hub.docker.com/r/testcontainers/helloworld image

### CODE REVIEW

File: modules/selenium/src/test/java/org/testcontainers/junit/BaseWebDriverContainerTest.java [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
         return driver;
     }
-
-    protected static void doSimpleExplore(BrowserWebDriverContainer rule) {
-        RemoteWebDriver driver = setupDriverFromRule(rule);
-        driver.get("http://en.wikipedia.org/wiki/Randomness");
-
-        // Oh! The irony!
-        assertTrue("Randomness' description has the word 'pattern'", driver.findElementByPartialLinkText("pattern").isDisplayed());
-    }
```
Comment (rnorth) [inline modules/selenium/src/test/java/org/testcontainers/junit/BaseWebDriverContainerTest.java:60]: There's no real sense in having two ways to test that Selenium is working, so just removed this altogether.

File: modules/selenium/src/test/java/org/testcontainers/junit/FlakyContainerCreationTest.java [hunk 2]
```diff
@@ -1,31 +0,0 @@
...
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.remote.RemoteWebDriver;
-import org.testcontainers.containers.BrowserWebDriverContainer;
-
-import java.io.File;
-
-/**
- * Created by rnorth on 30/01/2016.
- */
-public class FlakyContainerCreationTest {
```
Comment (rnorth) [inline modules/selenium/src/test/java/org/testcontainers/junit/FlakyContainerCreationTest.java:14]: This was old and ignored anyway, so have just removed.

File: modules/selenium/src/test/java/org/testcontainers/junit/LinkedContainerTest.java [hunk 3]
```diff
@@ -1,69 +0,0 @@
...
-import java.io.FileOutputStream;
-import java.io.PrintStream;
-
-import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
-
-
-/**
- * @author richardnorth
- */
-public class LinkedContainerTest {
```
Comment (rnorth) [inline modules/selenium/src/test/java/org/testcontainers/junit/LinkedContainerTest.java:26]: Given that all our tests will now use 'linked' (networked) containers, there's little point in having an explicit test for it that works another way.

**Review by bsideup — APPROVED** [review]

**Pull request merged**